src/displayapp/fonts/README.md

```diff
@@ -6,94 +6,40 @@
 
 ## Generate the fonts:
 
-* Open the [LVGL font converter](https://lvgl.io/tools/fontconverter)
-* Enter the settings for the font that you wish to convert
-* Click on Convert, download the file and place it in `src/DisplayApp/Fonts`
+* Make sure you have installed lv_font_conv, and it's in the PATH
+  * `npm i lv_font_conv`
```

Comment:
i'm not sure if adding npm instructions is necessary - or just annoying.

Reply:
I think this entire section can be removed, because you're not intended to run `generate.py` manually. Since `lv_font_conv` is now a build dependency, it should just be added [here](https://github.com/InfiniTimeOrg/InfiniTime/blob/develop/doc/buildAndProgram.md) with a link to the [repo](https://github.com/lvgl/lv_font_conv) with instructions being there.